tests/unit_tests/README.md

```diff
@@ -0,0 +1,36 @@
+# Run unit tests
+
+## on Linux
+
+1. Install NodeJS ([https://nodejs.org]()).
+1. Install `jq` and another necessary tools (e.g. `sed`).
+1. Open Terminal.
+1. Run `npm install`.
+1. Run `npm test`.
+
+## on Windows
+
+1. Install Windows Subsystem for Linux (WSL): https://docs.microsoft.com/en-us/windows/wsl/install-win10.
+1. Convert EOL in the script *./start_unit_tests.sh* from Windows (CR LF) to Unix (LF) - you can use the tool `dos2unix` in WSL to convert CR LF to LF.
+1. Follow the instructions for Linux. Install NodeJS and run the following commands in WSL.
```

Comment:
See the doxygen comment above

Reply:
Fixed in the commit [946a9a0b32ed95e838f5820b47c188f75b086223](https://github.com/polcak/jsrestrictor/pull/84/commits/946a9a0b32ed95e838f5820b47c188f75b086223).